.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
   - sudo sysctl -w vm.max_map_count=262144
   # Build images
   - docker login -u aschen -p $DOCKER_PASSWORD
-  - for dir in */; do base=$(basename "$dir"); if [ $base != 'samples' ] && [ $base != 'build' ]; then cd ${base}/docker && bash build_images.sh && bash push_images.sh && cd ../..; fi; done
+  - for dir in */; do base=$(basename "$dir"); if [ $base != 'samples' ] && [ $base != 'build' ] && [ $base != 'todo-with-kuzzle-and-vueJS' ]; then cd ${base}/docker && bash build_images.sh && bash push_images.sh && cd ../..; fi; done
```

Comment:
Since, this line become more and more complex. We should create shellscript to handle those actions. :wink:

Reply:
I can't agree more! 
Looks like you're the one who's going to do it @berthieresteban :)